workers.py

```diff
@@ -168,6 +171,33 @@ def query_sub_collection(
     return all_data
 
 
+def retrieve_hosts(host_inventory_url, b64_identity) -> dict:
+    headers = {"x-rh-identity": b64_identity}
+    url = host_inventory_url + '&page={}'
```

Comment:
What happens if an host is added into the inventory at the time we're listing this? We would get duplicit data on the second page? How do we handle this?

Reply:
`rad` would deal with duplicates